src/index.js

```diff
@@ -27,6 +28,10 @@ const injectDOM = function CozyBarInjectDOM ({lang, appName, iconPath}) {
   })
 }
 
+const getDefaultStackURL = function GetDefaultCozyURL () {
+  return document.querySelector('[role=application]').dataset.cozyDomain
```

Comment:
Ok my primary question about `[role=application]` was into this outdated comment @m4dz, sorry about that :)